bigquery/README.md

```diff
@@ -12,21 +12,24 @@ if you haven't done so already.
     bq --project_id MY_GOOGLE_CLOUD_PROJECT mk platinum_genomes
     bq --project_id MY_GOOGLE_CLOUD_PROJECT cp google.com:biggene:platinum_genomes.sample_info  MY_GOOGLE_CLOUD_PROJECT:platinum_genomes.sample_info
     ```
-  * Change project ids in `config/platinum_genomes/facet_fields.csv`.
+  * Change project ids in `dataset_config/platinum_genomes/facet_fields.csv`.
 * If you want to use your own dataset:
-  * Create a `config/private` directory. (Files in `config/private` won't be
-added to this git repo.) Make a new directory under `config/private` and copy
-`config/template/*` to it.
+  * For each dataset, there is a directory in `dataset_config`, e.g.
```

Comment:
Since no directory yet exists for each dataset, replace first two sentences with:

Create a directory for your dataset, e.g. `dataset_config/amp_pd`.